src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -152,7 +189,7 @@ private void updateIsInputStateByNameAndValue(IContainer containerToUpdate, ICon
          if (templateContainer == null)
             return;
 
-         foreach (var templateParameter in templateContainer.AllParameters(x => x.Visible && x.Editable && x.ValueIsComputable()))
```

Comment:
Some parameters are not editable such as dose. Not sure why and this should probably be changed
For the conversion, if parameters are visible and have changed, they should be Is_Default false. THis check was not required